src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/DynamicFeature.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement
+{
+    internal class DynamicFeature
+    {
+        [JsonPropertyName("id")]
+        public string Id { get; set; }
+
+        [JsonPropertyName("client_assigner")]
+        public string ClientAssigner { get; set; }
+
+        [JsonPropertyName("variants")]
+        public List<FeatureVariant> Variants { get; set; } = new List<FeatureVariant>();
+
+        // Reserved to allow variants list to be pulled from separate configuration.
```

Comment:
I don't see it being used anywhere, is that right? Reserving it isn't valuable if we don't process it.